pkg/processor/build/builder_test.go

```diff
@@ -738,7 +738,7 @@ func (suite *testSuite) testResolveFunctionPathArchive(buildConfiguration functi
 		httpmock.RegisterResponder("GET", archiveFileURL, responder)
 	}
 
-	path, err := suite.builder.resolveFunctionPath(buildConfiguration.Path)
+	path, _, err := suite.builder.resolveFunctionPath(buildConfiguration.Path)
```

Comment:
why not checking the `inferedcodeentrytype` as well?

Reply:
This is not relevant there because whenever this test function is called it is with code entry type (archive/s3/github) - and inferred type is not relevant when codeEntryType is explicitly given